integration/duckdb_lance/src/arrow.rs

```diff
@@ -107,8 +107,7 @@ pub fn to_duckdb_logical_type(data_type: &DataType) -> Result<LogicalType> {
             child.data_type(),
         )?))
     } else {
-        println!("Unsupported data type: {data_type}, please file an issue https://github.com/eto-ai/lance");
-        todo!()
+        todo!("Unsupported data type: {data_type}, please file an issue at https://github.com/lancedb/lance");
```

Comment:
`todo!`->`log::warn!`

Reply:
We are already panicking, so it's reasonable to keep panicking here.

Reply:
thanks, changed to `panic!`

Reply:
oh, I mean this line: https://github.com/lancedb/lance/blob/2377632a47c39211257dca5c05a5830e38e38fcd/rust/lance-index/src/vector/hnsw/builder.rs#L229